packages/core/test/client.test.ts

```diff
@@ -240,13 +252,14 @@ describe('@bugsnag/core/client', () => {
     })
 
     it('leaves a breadcrumb of the error', () => {
-      const payloads = []
+      const payloads: any[] = []
       const client = new Client({ apiKey: 'API_KEY_YEAH' })
       client._setDelivery(client => ({ sendEvent: (payload) => payloads.push(payload) }))
       client.notify(new Error('foobar'))
       expect(client._breadcrumbs.length).toBe(1)
       expect(client._breadcrumbs[0].type).toBe('error')
       expect(client._breadcrumbs[0].message).toBe('Error')
+      // @ts-ignore
```

Comment:
This one might actually point to a shortcoming in the Breadcrumb typings, or the data used in the test needs correcting. I wasn't sure, what do you think @bengourley ?

Reply:
Yeah looks to be the case. Previously breadcrumb metadata was only allowed to be 1 level deep – now it can be of arbitrary depth. Let's leave ignore in this PR but fix this in another PR.